packages/as-transformer/src/transformers/massaExportImports.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import {
+  IdentifierExpression,
+  ImportDeclaration,
+  ImportStatement,
+} from 'assemblyscript/dist/assemblyscript.js';
+import { Update, GlobalUpdates } from './interfaces/Update.js';
+// import { IExpressionTransformer } from './interfaces/IExpressionTransformer';
+
+/**
+ * The MassaExportCalls is a specific transformer that works by replacing a any massa exported function calls
+ * by their transformed version.
+ */
+export class MassaExportImports {
+  isMatching(expression: string): boolean {
```

Comment:
Missing doc:

Example:
```
  /**
   * Checks if a given expression matches with any of the functions exported using the "MassaExport" keyword.
   * 
   * @param expression - A string representing the expression to match.
   * 
   * @returns A boolean indicating whether the given expression matches any of the "MassaExport" functions.
   */
   ```

Reply:
On second thoughts, i'll do the documentation of this class in the appropriate issue, since this class is not used at the moment. If it's ok ofc.